generators/generator-base.js

```diff
@@ -2393,6 +2393,18 @@ module.exports = class extends PrivateBase {
         generator.experimental = context.configOptions.experimental || context.options.experimental;
     }
 
+
+    setupSpringServiceOptions(generator, context = generator) {
+        if (!context.options.name) {
+            if (context.options._) {
+                generator.name = context.options._;
+            } else {
+                this.argument('name', { type: String, required: true });
```

Comment:
The problem is without this line yeoman will not register the arg at all. may be there is a problem when composing the blueprint. Check how its done for entity gen, it also has name argument